api/db-cdr/generate-cdr/build-cb-criteria-surveys.sh

```diff
@@ -343,7 +343,7 @@ JOIN (
 WHERE domain_id = 'SURVEY'
 AND type = 'PPI'
 AND subtype = 'QUESTION'"
-questionConceptIds=$(bq --quiet --project_id="$BQ_PROJECT" query --nouse_legacy_sql --format csv "$query")
+questionConceptIds=$(bq --quiet --project_id="$BQ_PROJECT" query --nouse_legacy_sql --format csv -n 1000 "$query")
```

Comment:
It is better to use a constant for 1000 at the top of the file and reference it here, I think. NUM_PFFH_QUESTIONS=1000

Reply:
it's only used once in the file. I we need to reuse it we can make it a constant.